manage.py

```diff
@@ -808,6 +808,55 @@ def lint(filename, additional_properties):
     write_yaml_file(filename, elements)
 
 
+@cli.command()
+def update_sustainability_elements():
+    """Update mapping/sustainability.yaml"""
+
+    filename = basedir / 'mapping' / 'sustainability.yaml'
+
+    source = csv_reader(
```

Comment:
Can you add a comment with a link to the spreadsheet itself?

Reply:
Done in https://github.com/open-contracting/infrastructure/pull/425/commits/6b1a950834b09d16e0cefcdf643b43ad164b5105